cal/cal/models.py

```diff
@@ -583,20 +584,38 @@ def query(self, calendar_ids=None, start=None, end=None):
 
         return events_qs
 
-    def get_hours_per_week(self, calendar_ids=None, start=None, end=None):
+    def get_hours_per_week(self, timezone, calendar_ids=None, start=None, end=None):
         """
         Returns a list of week-hour tuples corresponding to the events in this ColorCategory.
         Each week starts at the start time.
         """
         week_hours = []
         events = self.query().order_by('start')
         i = 0
-        start = events[0].start
+        # Convert start to local time
+        start = handle_time_string(str(events[0].start), timezone)
+        # Change start date to be Monday beginning of week
+        while start.weekday() != 0:
+            start = start - relativedelta(days=1)
```

Comment:
Sure I'll change it for this but I need it for `end = start + relativedelta(days=7)` because I'm going to be generalizing it for months and relativedelta makes sure that adding to months will always start on the same day.